### PR TITLE
Use ACP profile image dimension setting in cropper

### PR DESCRIPTION
**Pull request by pichalite**
* Use ACP profile image dimension setting in cropper to restrict uploaded image dimension.
* Fix aspect ratio setting.

### CODE REVIEW

File: public/src/client/account/edit.js [hunk 1]
```diff
@@ -216,10 +218,13 @@ define('forum/account/edit', ['forum/account/header', 'translator', 'components'
 
 			pictureCropper.show({
 				socketMethod: 'user.uploadCroppedPicture',
-				aspectRatio: '1 / 1',
+				aspectRatio: 1 / 1,
```
Comment (julianlam) [inline public/src/client/account/edit.js]: resolves to `Number` type, is this intentional?

File: public/src/client/account/edit.js [hunk 2]
```diff
@@ -242,17 +247,20 @@ define('forum/account/edit', ['forum/account/header', 'translator', 'components'
...
 						uploadModal.modal('hide');
-						
+
 						pictureCropper.handleImageCrop({
 							url: url,
 							socketMethod: 'user.uploadCroppedPicture',
 							aspectRatio: '1 / 1',
+							allowSkippingCrop: false,
+							restrictImageDimension: true,
+							imageDimension: ajaxify.data.profileImageDimension,
```
Comment (julianlam) [inline public/src/client/account/edit.js]: Does this handle cases where `profileImageDimension` is not set?

File: public/src/client/account/edit.js (hunk 1 above)
Comment (pichalite) [inline public/src/client/account/edit.js]: Yes actually cropper js module requires a number. So, this is a bug fix.

File: public/src/client/account/edit.js (hunk 2 above)
Comment (pichalite) [inline public/src/client/account/edit.js]: It's handled on the API side. Returns 128 as default if not set in ACP. Part of this PR. 

https://github.com/pichalite/NodeBB/blob/fd0f0beaf22b300ca410551e0469a84b8af094e9/src/controllers/accounts/edit.js#L30

**Comment by julianlam** [comment]
Thank you 😄

**Pull request merged**